### PR TITLE
Kabisa theming

### PR DESCRIPTION
**Pull request by alianza**
Added Kabisa theming throughout the application

- Logo's
- Favicons
- SEO
- Links
- Colors

### CODE REVIEW

**Comment by alianza** [comment]
⚡️ [Lighthouse report](https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1625486167871-97733.report.html) for the changes in this PR:
| Category | Score |
| --- | --- |
| 🟠 Performance | 86 |
| 🟢 Accessibility | 100 |
| 🟢 Best practices | 100 |
| 🟢 SEO | 92 |
| 🟠 PWA | 83 |
 
*Lighthouse ran on [https://headless-commerce-poc-h4m0l5hq4-alianza.vercel.app/](https://headless-commerce-poc-h4m0l5hq4-alianza.vercel.app/)*

File: components/common/Footer/Footer.tsx [hunk 1]
```diff
@@ -31,12 +32,12 @@ const Footer: FC<Props> = ({ className, pages }) => {
       <Container>
         <div className="grid grid-cols-1 lg:grid-cols-12 gap-8 border-b border-accent-2 py-12 text-primary bg-primary transition-colors duration-150">
           <div className="col-span-1 lg:col-span-2">
-            <Link href="/">
+            <Link href={"https://kabisa.nl/"}>
```
Comment (egonm12) [inline components/common/Footer/Footer.tsx:35]: Very nitpicky, try to remove the curly braces when you are setting a string as a prop

```suggestion
            <Link href="https://kabisa.nl/">
```

File: components/common/Footer/Footer.tsx [hunk 2]
```diff
@@ -31,12 +32,12 @@ const Footer: FC<Props> = ({ className, pages }) => {
...
               <a className="flex flex-initial items-center font-bold md:mr-24">
-                <span className="rounded-full border border-accent-6 mr-2">
+                <span className="mr-2">
                   <Logo />
                 </span>
-                <span>ACME</span>
+                <span>kabisa</span>
```
Comment (egonm12) [inline components/common/Footer/Footer.tsx:40]:
```suggestion
                <span>Kabisa</span>
```

**Review by egonm12 — APPROVED** [review]
Two small comments, approved nevertheless so you can merge when you resolved the feedback.

File: components/common/Footer/Footer.tsx (hunk 1 above)
Comment (alianza) [reply components/common/Footer/Footer.tsx:35]: Noted & fixed :)

File: components/common/Footer/Footer.tsx (hunk 2 above)
Comment (alianza) [reply components/common/Footer/Footer.tsx]: Fixed :)

**Pull request merged**